### PR TITLE
update 8.0 with recent 7.0 changes, update travis config

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 [![Build Status](https://travis-ci.org/OCA/account-analytic.svg?branch=8.0)](https://travis-ci.org/OCA/account-analytic)
 [![Coverage Status](https://coveralls.io/repos/OCA/account-analytic/badge.png?branch=8.0)](https://coveralls.io/r/OCA/account-analytic?branch=8.0)
 
-OCA analytic accounting modules for Odoo
-========================================
+
+OpenERP / Odoo Analytic Accounting
```
Comment (pedrobaeza) [inline README.md]: I see no problem in previous text, and I find it more suitable, because it indicates that these are OCA modules.

**Comment by pedrobaeza** [comment]
Thanks for the changes.

:+1:

**Comment by sbidoul** [comment]
@gurneyalex would you merge 7.0 in 8.0 and rebase this PR?

**Comment by sbidoul** [comment]
This one is getting old, and is superseded by #17

**Comment by pedrobaeza** [comment]
I close then and continue on #17

**Pull request closed without merging**